spk/synocli-net/Makefile

```diff
@@ -17,7 +17,7 @@ MAINTAINER = ymartin59
 DESCRIPTION = "SynoCli Network Tools provides a set of small command-line utilities: screen, tmux, mosh, socat, nmap, sshfs, fritzctl. Credits to Sebastian Schmidt \(publicarray\) for icons"
 DISPLAY_NAME = SynoCli Network Tools
 STARTABLE = no
-CHANGELOG = "3. Build sshfs 2.10 with fuse updated to v2.9.9"
+CHANGELOG = "built with go 1.13.1"
 
 HOMEPAGE   = https://github.com/SynoCommunity/spksrc/wiki/FAQ-SynoCliNet
 LICENSE    = Mixed
```

Comment:
If you are updating this file:
I'd rather set LICENSE = Each tool is licensed unter its respective license.
_There does not exist a "mixed" license._

Reply:
Changed in https://github.com/SynoCommunity/spksrc/pull/3779/commits/2e581e055375d7545188c1ee72151e71ac3570d9